src/d3a/d3a_core/area_serializer.py

```diff
@@ -145,7 +147,9 @@ def optional(attr):
                     transfer_fee_const=grid_fee_constant,
                     external_connection_available=external_connection_available,
                     baseline_peak_energy_import_kWh=baseline_peak_energy_import_kWh,
-                    baseline_peak_energy_export_kWh=baseline_peak_energy_export_kWh)
+                    baseline_peak_energy_export_kWh=baseline_peak_energy_export_kWh,
```

Comment:
No need to do it now, but there could be a namedtuple that groups these 4 parameters (baseline_peak_energy_import/export, import/export_capacity) and use only the tuple as a constructor argument of the Area class. That way the number of constructor arguments of the area class will be decreased.